src/app/services/securedStorage.service.ts

```diff
@@ -64,7 +64,7 @@ export class SecuredStorageService {
     }
 
     async getIdentityData(): Promise<any> {
-        let identity = {};
+        const identity = {};
```

Comment:
is it right to use const if you are changing the state of the variable?

Reply:
Yes, I think so. What's `const` is the reference to the object instance. Then the object can mutate.

Reply:
That is, it means that `identity` won't be reassigned.